### ISSUE TITLE
When serving triples, Quilt/librdf does not merge graphs and coalesce triples properly

### ISSUE THREAD

**Issue by cgueret** [bbcarchdev/quilt#44]
When triples (rather than quads) are generated by Quilt, `librdf`'s serialisation does not merge the graphs into a single graph (removing duplicate triples along the way). This means that groups of triples which appear in multiple graphs are not coalesced into a single triple.

See example screen shot of http://acropolis.org.uk/0bbd7c84512541368e55a5c0756531ac

![screen shot 2016-04-15 at 14 20 09](https://cloud.githubusercontent.com/assets/196436/14562552/3f2bdd44-0315-11e6-9e20-f02bc35bf1ff.png)

**Comment by nevali** [comment]
The only sensible fix would be, when triples are being served, to generate a new `librdf_model` for serialisation; all of the source triples would then be streamed into the default graph of this model.

Whether this is worth the effort or not is open to debate, given that consumers should never choke on duplicate triples.

**Comment by cgueret** [comment]
I agree this is not impairing functionalities and that consumers should not choke on duplicate triples but, IMHO, those duplicates do not leave a good impression of the platform. This should probably be fix so that consumers can be more confident in using Quilt.
